package/cfg/rke2-cis-1.7-hardened/master.yaml

```diff
@@ -192,7 +192,7 @@ groups:
 
       - id: 1.1.17
         text: "Ensure that the controller-manager.conf file permissions are set to 600 or more restrictive (Automated)"
-        audit: "stat -c permissions=%a /var/lib/rancher/rke2/server/cred/controller.kubeconfig"
+        audit: "/bin/sh -c 'if test -e $controllermanagerkubeconfig; then stat -c permissions=%a $controllermanagerkubeconfig; fi'"
```

Comment:
You will need to also add a kubeconfig field under the `controllermanager` sectuib in `config.yaml` for this one to work.